packages/backend/package.json

```diff
@@ -12,6 +12,7 @@
     "typecheck": "tsc --noEmit",
     "build": "tsc",
     "test": "mocha",
+    "coverage": "nyc --reporter=lcov --reporter=text mocha",
```

Comment:
In the future we might want to add such scripts to all packages and have a repository level script too.